charts/kms/0.1.0/questions.yaml

```
@@ -0,0 +1,228 @@
+groups:
+  - name: "Container Image"
+    description: "configure container image"
+  - name: "Configuration"
+    description: "additional container configuration"
+  - name: "Networking"
+    description: "Networking / service configuration"
+  - name: "Storage"
+    description: "configure app volume mounts"
+  - name: "Ingress"
+    description: "Ingress configuration"
+
+portals:
+  web_portal:
+    protocols:
+      - "http"
+    host:
+      - "$node_ip"
+    ports:
+      - "$variable-service.port.port"
```

is `port.port` intentional here

```suggestion
      - "$variable-service.port.nodePort"
```